Userland/Libraries/LibC/sys/mman.cpp

```diff
@@ -90,6 +90,12 @@ int mlock(const void*, size_t)
     return 0;
 }
 
+int munlock(const void*, size_t)
```

Comment:
```suggestion
// https://pubs.opengroup.org/onlinepubs/9699919799/functions/munlock.html
int munlock(const void*, size_t)
```

Reply:
Totally forgot we started this! Will add them everywhere, thanks